eng/Versions.props

```diff
@@ -29,7 +29,7 @@
     <VisualStudioEditorNewPackagesVersion>17.0.487</VisualStudioEditorNewPackagesVersion>
     <ILAsmPackageVersion>5.0.0-alpha1.19409.1</ILAsmPackageVersion>
     <ILDAsmPackageVersion>5.0.0-preview.1.20112.8</ILDAsmPackageVersion>
-    <MicrosoftVisualStudioLanguageServerProtocolPackagesVersion>17.1.11</MicrosoftVisualStudioLanguageServerProtocolPackagesVersion>
+    <MicrosoftVisualStudioLanguageServerProtocolPackagesVersion>17.2.3</MicrosoftVisualStudioLanguageServerProtocolPackagesVersion>
```

Comment:
📝 I believe this will need to go in a vs-deps branch

Reply:
@jmarolf can you help me with this?

Reply:
I think LSP will not work (since now we rely on binding redirects in VS) when F5'ing roslyn in older VS.  But integration tests should still work.  It might not be necessary to create a -vs-deps for this and just rely on having 17.2 for LSP related changes.

Reply:
And LSP in general should be backwards compatible, so it might be useful here in the future to just add binding redirects for the newer version when we F5 here - https://github.com/dotnet/roslyn/blob/main/src/VisualStudio/Setup.Dependencies/Roslyn.VisualStudio.Setup.Dependencies.csproj#L23